qupath-core/src/main/java/qupath/lib/objects/utils/ObjectMerger.java

```diff
@@ -570,6 +680,74 @@ private static PathObject mergeObjects(List<? extends PathObject> pathObjects) {
         return mergedObject;
     }
 
+    private static MeasurementList mergeMeasurements(List<? extends PathObject> pathObjects, MeasurementStrategy measurementStrategy) {
+        if (pathObjects.isEmpty()) {
+            return null;
+        }
+        return switch(measurementStrategy) {
+            case IGNORE -> null;
+            case USE_FIRST -> pathObjects.getFirst().getMeasurementList();
+            case MEAN -> {
+                var out = MeasurementListFactory.createMeasurementList(
+                        pathObjects.getFirst().getMeasurementList().size(),
+                        MeasurementList.MeasurementListType.FLOAT
+                );
+                for (var es: pathObjects.getFirst().getMeasurements().entrySet()) {
+                    out.put(
+                            es.getKey(),
+                            pathObjects.stream().mapToDouble(po -> po.getMeasurementList().get(es.getKey())).sum() / pathObjects.size());
```

Comment:
Not `DoubleStream.average()`?
Should we do anything with `NaN` here? I guess not... but I wonder if we might ever end up generating an area and a line, and merge the line quite unnecessarily and end up killing some otherwise valid measurements with NaN. Hopefully not...